### PR TITLE
z1338 - fix server name error

### PR DESCRIPTION
**Pull request by vnarek**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ zcli.config.yml
 zcli.data
 tools/npm/node_modules
 .vscode
+zerops.yml
```
Comment (kawaiiyoshiyuki) [inline .gitignore:6]: we might probably ignore both zerops and import yaml files regardless there position like 
```
*zerops.yml
*import.yml
```

File: src/proto/business/client.go [hunk 2]
```diff
@@ -56,12 +57,25 @@ func (h *Handler) CreateClient(ctx context.Context, grpcApiAddress string, token
 
 }
 
+const serverNamePrefix = "zbusinessapi"
+
+func verifyPeerServerName(p *advancedtls.VerificationFuncParams) (*advancedtls.VerificationResults, error) {
+	if p.Leaf == nil {
+		return nil, fmt.Errorf("missing leaf certificate")
+	}
+	for _, name := range p.Leaf.DNSNames {
```
Comment (kawaiiyoshiyuki) [inline src/proto/business/client.go:66]: I am unsure how this works so I trust you :D

**Review by kawaiiyoshiyuki — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (vnarek) [reply .gitignore:6]: that's a good idea :)

File: src/proto/business/client.go (hunk 2 above)
Comment (vnarek) [reply src/proto/business/client.go:66]: Thanks :D. It just basically compares prefixes of the domains inside a certificate and not the exact match.

**Pull request merged**